evm.md

```diff
@@ -526,19 +511,11 @@ The arguments to `PUSH` must be skipped over (as they are inline), and the opcod
  // ------------------------------------------
     rule <k> #pc [ OP ] => . ... </k>
          <pc> PCOUNT => PCOUNT +Int #widthOp(OP) </pc>
-      requires notBool isJumpOp(OP)
-
-    rule <k> #pc [ OP ] => . ... </k>
```

Comment:
I think you have an off by one error here. You're going to skip executing the jumpdest instruction. Maybe that's fine, since JUMPDEST doesn't do anything, but it seems suspicious to me.

Reply:
`#endBasicBlock` consumes everything up until the next `#execute`, so `#pc` isn't executed when `JUMP` affects the `<pc>` cell.

Reply:
We definitely need to execute the `JUMPDEST` instruction when we arrive at it, because its gas needs to be deducted. If we were skipping it, no way we wolud pass tests.